packages/core/src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.spec.ts

```diff
@@ -457,6 +457,23 @@ describe('StripeUPEPaymentStrategy', () => {
                         expect(response).toBe(store.getState());
                     });
 
+                    it('with a guest user with postal code', async () => {
+                        jest.spyOn(store.getState().customer, 'getCustomer')
+                            .mockReturnValue(undefined);
+                        jest.spyOn(store.getState().shippingAddress, 'getShippingAddress')
+                            .mockReturnValue({
+                                ...getShippingAddress(),
+                                postalCode: '12345',
+                            });
+
+                        const response = await strategy.execute(getStripeUPEOrderRequestBodyMock());
+
+                        expect(orderActionCreator.submitOrder).toHaveBeenCalled();
```

Comment:
Do we need to test the payload if it has postalCode as part of the address?

Reply:
Yes, we do it because we need to check if the Postal Code is already sent in the address in case it is not sent we add it to the stripe element.